pkg/build/api/validation/validation_test.go

```diff
@@ -67,7 +63,7 @@ func TestBuildValidationFailure(t *testing.T) {
 			Phase: buildapi.BuildPhaseNew,
 		},
 	}
-	if result := ValidateBuild(build); len(result) != 3 {
+	if result := ValidateBuild(build); len(result) != 2 {
```

Comment:
Hmm... the compiler "checks" that `ValidateBuild` returns a list of errors. IMO this test as-is adds no value, just more LOC.

If it was about returning multiple errors, the condition would be `len(results) > 1`, not some hard coded magic value.

If we had only that test we could easily break validation without ever noticing.